examples/ReactExample/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Feb 10 01:35:36 GMT 2016
+#Mon Apr 08 14:33:38 CEST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
```

Comment:
shouldn't it be 
`distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip`
?